app/controllers/places_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PlacesController < ApplicationController
+  def index
+    @places = Place.all
+    render json: @places
+  end
+
+  def show
+    @place = Place.find(params[:id])
```

Comment:
`@place = Place.find(params[:id])` repeat a lot. Move it to `before_action` and set it for actions where this line is used